prow/plugins/golint/golint.go

```diff
@@ -70,17 +70,8 @@ type githubClient interface {
 	ListPullRequestComments(org, repo string, number int) ([]github.ReviewComment, error)
 }
 
-const defaultConfidence = 0.8
-
-func minConfidence(g *plugins.Golint) float64 {
```

Comment:
I'd be fine with some helper function like this living in the config package.

What I don't like is the entry to the plugin -- which runs on every webhook for repos the plugin is enabled -- having a `*whatever.SomethingThatCanBeNull.SomethingElseThatCanBeNull` rather than having handle deal with a float